virt-who-config.5

```diff
@@ -249,6 +249,27 @@ Absolute path to the JSON file that has the same structure as file returned from
 .br
 }
 
+.SH CONFIGURATION MIGRATION
+
+Previous versions of virt-who employed other configuration means based on environment variables. The first was system-wide environment variables. The second was a file [/etc/sysconfig/virt-who]
```

Comment:
I guess that the 'means' word was added by mistake in the first sentence, right?